node/src/Pos/Aeson/Types.hs

```diff
@@ -2,20 +2,32 @@ module Pos.Aeson.Types
        (
        ) where
 
-import           Data.Aeson       (ToJSON (toJSON), object, (.=))
+import           Universum
+
+import           Data.Aeson       (FromJSON (..), ToJSON (toJSON), object, withObject,
+                                   (.:), (.=))
 import           Data.Aeson.TH    (defaultOptions, deriveToJSON)
 import           Formatting       (build, sformat)
 
 import           Pos.Aeson.Crypto ()
+import           Pos.Core.Address (decodeTextAddress)
 import           Pos.Txp          (TxOut (..))
-import           Pos.Types        (coinToInteger)
+import           Pos.Types        (coinToInteger, integerToCoin)
+import           Pos.Util.Util    (eitherToFail)
 import           Pos.Web.Types    (GodTossingStage)
 
 instance ToJSON TxOut where
     toJSON TxOut{..} = object [
         "coin"    .= coinToInteger txOutValue,
         "address" .= sformat build txOutAddress ]
 
+
+instance FromJSON TxOut where
+    parseJSON = withObject "TxOut" $ \o -> do
+        txOutValue   <- eitherToFail . integerToCoin =<< o .: "coin"
+        txOutAddress <- eitherToFail . decodeTextAddress =<< o .: "address"
```

Comment:
Almost the entire module [is gone](https://github.com/input-output-hk/cardano-sl/blob/master-release-candidate/node/src/Pos/Aeson/Types.hs#L15) in `master`. @pva701 , does this ring a bell? Ah, I think it was introduced as part of https://github.com/input-output-hk/cardano-sl/commit/fee01810a33d73ebf349a0995efc0dd74ced6f58#diff-314075405d88168e3c4ea2e011625453